File: Java/프로그래머스/0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int result=-1;
        String numStr = Integer.toString(num);
        int[] numArray = new int[numStr.length()];
        for (int i = 0; i < numStr.length(); i++) {
            numArray[i] = Character.getNumericValue(numStr.charAt(i));
        }
        for(int i=0 ; i<numArray.length ; i++){
            if(numArray[i]==k){
                result=i+1;
                break;}
        }
        int answer = result;
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120904. 숫자 찾기/숫자 찾기.js
function solution(num, k) {
    var answer = -1;
    var num1 = String(num);
    var k1 = String(k);
    for(var i=0; i<num1.length; i++){
        if(num1.charAt(i)==k1) {
            answer=i+1;
            break;
        }
    }
    return answer;
}